libtenzir/builtins/operators/every_cron.cpp

```diff
@@ -393,10 +393,59 @@ class every_plugin2 final : public virtual operator_factory_plugin {
   }
 };
 
+class cron_plugin2 final : public virtual operator_factory_plugin {
+public:
+  auto name() const -> std::string override {
+    return "tql2.cron";
+  }
+
+  auto
+  make(invocation inv, session ctx) const -> failure_or<operator_ptr> override {
+    auto interval = located<std::string>{};
+    auto pipe = pipeline{};
+    TRY(argument_parser2::operator_(name())
+          .positional("interval", interval)
```

Comment:
It's not really an interval. The docs previously used "expression" because it's a cron expression (not a TQL expression).

Reply:
I'm aware but `expresssion:string` might look confusing.

Reply:
Maybe `specification` or `spec`?